src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -117,7 +117,9 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution, dnnl::convo
         auto cldnn_prim = arg.get_primitive();
         auto weights_layout = arg.get_dependency(1).get_output_layout();
         auto grouped_weights = format::is_grouped(weights_layout.format) || arg.get_primitive()->grouped_weights_shape;
-        cldnn::format out_fmt = onednn::convert_format(onednn::get_format_by_desc(pd.weights_desc(0)), grouped_weights);
+        auto onednn_desc = onednn::get_format_by_desc(pd.weights_desc(0));
+        cldnn::format out_fmt = (onednn_desc != dnnl::memory::format_tag::undef) ? onednn::convert_format(onednn_desc, grouped_weights)
+                                                                                    : onednn::find_format(pd.weights_desc(0), grouped_weights);
```

Comment:
What about changing as below and do everything within find_format?
```suggestion
        cldnn::format out_fmt = onednn::find_format(pd.weights_desc(0), grouped_weights);
```

Reply:
Agree, updated it.